mutiny-core/src/nodemanager.rs

```diff
@@ -1052,14 +1055,115 @@ impl<S: MutinyStorage> NodeManager<S> {
             return Err(MutinyError::WalletOperationFailed);
         };
 
+        // If we are in safe mode, we don't create payjoin sessions
```

Comment:
I think we can do this anyways in safe mode given that it's mostly lightning related for that.